### PR TITLE
.travis.yml: Xcode 7.3, one OS X version, cleanup.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
CC @xu-cheng

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,14 +18,8 @@ before_install:
   - rsync -az "$TRAVIS_BUILD_DIR" "$(brew --repo $TRAVIS_REPO_SLUG)"
   - export TRAVIS_BUILD_DIR="$(brew --repo $TRAVIS_REPO_SLUG)"
   - cd "$(brew --repo $TRAVIS_REPO_SLUG)"
-  - env | grep TRAVIS_
 
 install:
```
Comment (xu-cheng) [inline .travis.yml]: This line can be removed as well.

Comment (mikemcquaid) [inline .travis.yml]: Which one, sorry? the `install:`?

Comment (xu-cheng) [inline .travis.yml]: Yep. I think there is no need for a separate install section.

Comment (mikemcquaid) [inline .travis.yml]: Feels a bit cleaner to me given the docs so will just leave it as-is for now.

**Pull request merged**